Dshop/apps/shipping/templates/shipping/shipping_form.html

```diff
@@ -0,0 +1,57 @@
+<!-- your_template.html -->
+
+{% extends 'base.html' %}
+
+{% block content %}
+<br>
+{% if error %}
+<p>{{ error }}</p>
+{% else %}
+<form method="post" action="{% url 'handle-method-choice' %}" class="container mt-4" id="shippingForm">
+    {% csrf_token %}
+
+    <div class="field">
```

Comment:
This is not handling any errors. Why not using easy approach as form.as_p or somilar? Or at least support errors as described in documentation: https://docs.djangoproject.com/en/5.0/topics/forms/#rendering-fields-manually
However, I would suggest using: https://docs.djangoproject.com/en/5.0/topics/forms/#reusable-form-templates if you really insist on custom HTML.

Reply:
Actually, I see it is totally missing a form now, so I think this is simple test right?
If so, ok. 
In the long run, I would prob make a model to handle delivery with all the fields above. Then we can use form to easy add delivery to order and we have whole data in DB for audit and client history